src/main/java/com/prgrms/vouchermanagement/voucher/io/VoucherConsole.java

```diff
@@ -0,0 +1,85 @@
+package com.prgrms.vouchermanagement.voucher.io;
+
+import java.text.MessageFormat;
+import java.util.List;
+import java.util.Scanner;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.prgrms.vouchermanagement.voucher.domain.Voucher;
+
+@Component
+public class VoucherConsole implements VoucherInput, VoucherOutput {
+
+	private final Scanner sc = new Scanner(System.in);
+	private final Logger logger = LoggerFactory.getLogger(VoucherConsole.class);
+
+	@Override
+	public void showMenu() {
+		System.out.println("==== Voucher Program ====");
+		System.out.println("Type exit to exit program.");
+		System.out.println("Type create to create a new voucher.");
+		System.out.println("Type list to list all vouchers.");
+	}
+
+	@Override
+	public long inputVoucherInfo() {
+		try {
+			String input = sc.nextLine();
+
+			return Long.parseLong(input);
+		} catch (NumberFormatException e) {
+			failCreation();
+
+			throw new IllegalArgumentException("VoucherDiscountInfo must be numeric", e);
+		}
+	}
+
+	@Override
+	public String selectVoucherType() {
+		return sc.nextLine();
+	}
+
+	@Override
+	public String selectedMenu() {
+		return sc.nextLine();
+	}
+
+	@Override
+	public void notifyNoMappingSelection() {
+		System.out.println("일치하는 항목이 존재하지 않습니다");
+	}
+
+	@Override
+	public <T> void showAll(List<T> list) {
+		if (list.isEmpty())
+			System.out.println("비어있습니다");
+
+		list.stream().forEach(System.out::println);
```

Comment:
```suggestion
		list.forEach(System.out::println);
```

list는 바로 forEach()문이 가능합니다.